examples/gno.land/r/x/grc20_dynamic_call/baz/baz.gno

```diff
@@ -0,0 +1,130 @@
+package baz
+
+import (
```

Comment:
I suggest using either a bar or baz instead of exposing public methods. Create a safe object with the following code:

```go
var baz = grc20.NewAdminToken(...)
var Baz = baz.GRC20()
// call Baz.Approve, etc directly
```

This recommendation reduces the codebase and eliminates redundancy. It demonstrates that the registry can function with standard public-facing grc20 implementations or grc20 objects. Ideally, keep a single foo file with public methods, while keeping the other files around 30 lines long. This approach will make the example more powerful and concise.

Reply:
calling `Baz.Transfer()` does panic `VM call panic: cannot modify external-realm or non-realm object` because of updating its token balance.

as @tbruyelle [mentioned](https://github.com/gnolang/gno/pull/1275#issuecomment-1888839233), #1257 fix need to be settle down for this pr to use grc20 object instead of public methods